doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -655,17 +801,16 @@ This prevents the CPU from executing any instructions until a debugger manually
 Protecting the device
 *********************
 
-To protect the nRF54H20 SoC in a production-ready device, you must enable the following UICR-based security mechanisms:
+To protect the nRF54H20 SoC in a production-ready device, you must enable the following UICR-based security mechanisms.
+For information on how to configure these UICR settings, see :ref:`ug_nrf54h20_ironside_se_uicr`.
 
-* UICR.APPROTECT - Disables all debug and AP access.
+* :kconfig:option:`CONFIG_GEN_UICR_APPROTECT_APPLICATION_PROTECTED` - Disables all debug and AP access.
   It restricts debugger and access-port (AP) permissions, preventing unauthorized read/write access to memory and debug interfaces.
-* UICR.LOCK - Freezes non-volatile configuration registers.
+* :kconfig:option:`CONFIG_GEN_UICR_LOCK` - Freezes non-volatile configuration registers.
   It locks the UICR, ensuring that no further UICR writes are possible without issuing an `ERASEALL` command.
-* UICR.PROTECTEDMEM - Enforces integrity checks on critical code and data.
+* :kconfig:option:`CONFIG_GEN_UICR_PROTECTEDMEM` - Enforces integrity checks on critical code and data.
   It defines a trailing region of application-owned MRAM whose contents are integrity-checked at each boot, extending the root of trust to your immutable bootloader or critical data.
-* UICR.MPCCONF - Configures memory protection for the bootloader region.
-  It should be used to set RX-only (read and execute) permissions on the PROTECTEDMEM region containing the bootloader, preventing unauthorized modification while allowing execution.
-* UICR.ERASEPROTECT - Prevent bulk erasure of protected memory.
+* :kconfig:option:`CONFIG_GEN_UICR_ERASEPROTECT` - Prevent bulk erasure of protected memory.
```

Comment:
Not part of this PR I know, but just wondering if "bulk erasure" is a term we use?

cc @FrancescoSer